03b_model/src/run_model.py

```diff
@@ -291,34 +581,183 @@ def plot_save_predictions(trainval_df, out_dir, run_id):
     
     #save predictions
     trainval_df.to_csv(os.path.join(out_dir, run_id, 'ModelResults.csv'))
+    
+def run_replicates(n_reps, prepped_model_io_data_file):
+    '''
+    run model replicates to understand variability
```

Comment:
Suggested add to description: and saves the results and error plots to sub-directories in run_id